builder

```diff
@@ -79,6 +80,9 @@ touch $lockfile
 log "fetching from repo"
 git fetch --quiet
 log "checking out ${BRANCH}"
+git checkout ${BRANCH} > /dev/null 2>&1
```

Comment:
My thought here is that if you are running the builder a second time from the same dir there is no way to tell if you are on the same you are attempting to merge into to tell what has changed... doing this allows the code below to tell what commit the code is currently on... 

Hopefully that makes sense.